auth/native/data.go

```diff
@@ -15,8 +30,17 @@ need to use the following token to finish this process:
 {{.Token}}
 
 If you think this is email is wrong, just ignore it.`))
-
-var passwordResetConfirm = template.Must(template.New("reset").Parse(`Subject: [tsuru] Password successfully reset
+}
+
+func getEmailResetPasswordSucessfullyTemplate() (*template.Template) {
```

Comment:
The `error` should not be ignored here. You should change this function to return `Template` and `error`.